utils/docker/run-build.sh

```diff
@@ -167,3 +167,14 @@ cd ..
 rm -rf build
 
 rm -r $INSTALL_DIR
+
+mkdir doc_update
+
+# Trigger auto doc update on master
+if [[ "$AUTO_DOC_UPDATE" == "1" && "$TRAVIS_BRANCH" == "master" && "$TRAVIS_PULL_REQUEST" == "false" ]]; then
+	echo "Running auto doc update"
+	utils/docker/run-doc-update.sh
+fi
+
+cd ..
+rm -rf doc_update
```

Comment:
Something is not right here. You never change current directory into doc_update, so rm will not find this directory.